<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>数据处理</string>
      </property>
      <widget class="QTextEdit" name="dataSend">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="dataReceived">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>261</width>
         <height>571</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>57</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>发送数据</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>560</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空接收数据</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>串口配制</string>
      </property>
      <widget class="QPushButton" name="ScanButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>扫描</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
      <widget class="QPushButton" name="OpenSerialButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="currentText">
        <string>600</string>
       </property>
       <item>
        <property name="text">
         <string>600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>460800</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="ParityBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>无校验</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>奇校验</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>偶校验</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0校验</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1校验</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>控制流</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="BitBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="PortBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="StopBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>320</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="ControlBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
